DemoServer/AdditionalLanguages/nodejs/demo/queries/projectingUsingFunctions/metadata.json

```diff
@@ -0,0 +1,120 @@
+                "  * formatName -  process the 'FirstName' & 'LastName' document fields.<br>",
+                "  * formatTitle - process the document 'Title' field."
+            ],
+            "Assets": [
+                {
+                    "Title": "Projection with declared function",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/nodejs/client-api/session/querying/how-to-project-query-results#example-iv---projection-with-"
+                }
+
+            ]
+        },
+        {
+            "Title": "Define the RQL query",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Within the raw-query-string, define the `RQL query` that will be executed on the server.",
+                "",
+                "* Use `select` and call the JavaScript function defined from the previous step.",
+                "",
+                "* In the above RQL, a query is defined on the 'Employees' collection.",
+                "  Note the _select_ usage so that returned results are structured according to the defined _output_ method."
+            ],
+            "Assets": [
+                {
+                    "Title": "What is RQL",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/5.3/nodejs/indexes/querying/what-is-rql"
+                }
+
+            ]
+        },
+        {
+            "Title": "Set the query",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Pass the raw-query-string to the Session `rawQuery()` method."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type": "demo",
+                    "Url": "/nodejs/queries/query-overview"
+                },
+                {
+                    "Title": "Projection with declared function",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/nodejs/client-api/session/querying/how-to-project-query-results#example-iv---projection-with-"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* The 'projectedQueryWithFunctions' variable only stores the query definition.<br>",
+                "  Execute the query by calling `all()`.",
+                "",
+                "* The _projectedResults_ variable will hold the resulting matching entities.",
+                "",
+                "* The output projected entities returned by the query are Not tracked by the Session,<br>",
+                "  as they are not full documents.",
+                "",
+                "* Any change made to these entities will Not be sent to the server upon calling SaveChanges()."
```

Comment:
SaveChanges() -> saveChanges() - this is node.js

Reply:
Done